include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3568,6 +3626,14 @@ ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_CUDA, _In_ OrtSessionOpt
 */
 ORT_API_STATUS(OrtSessionOptionsAppendExecutionProvider_MIGraphX, _In_ OrtSessionOptions* options, int device_id);
 
+/*
```

Comment:
This function should be deleted. The other provider ones like it were left for backwards compatibility. New provider functionality should be done only through the OrtApi as it is versioned.

Reply:
Got it